api/app.py

```diff
@@ -143,6 +178,54 @@ def predict():
     # return cats[text_clf.predict([text])[0]]
     return cats[gs_clf.predict([text])[0]]
 
+
+# Request format is /predictBatch?SpareMeElement0=<encoded_string>&id_token=<token>
+@app.route('/predictBatch', methods=['GET'])
```

Comment:
I think we will want to use POST requests for everything once SSL is working in order to prevent arguments leaking out of the encryption into server logs and stuff, not to mention the possibility of hitting the relatively low capacity limit of GET args